Mathlib/NumberTheory/NumberField/Basic.lean

```diff
@@ -129,7 +188,7 @@ protected noncomputable def equiv (R : Type*) [CommRing R] [Algebra R K]
 
 variable (K)
 
-instance : CharZero (𝓞 K) :=
+instance [CharZero K] : CharZero (𝓞 K) :=
```

Comment:
It does not since quite a longtime (maybe the beginning?). But note that the notation `𝓞 K` is scoped, so this is not a problem if you want that `𝓞 ℚₚ = ℤₚ`.